docs/source/index.rst

```diff
@@ -84,17 +38,34 @@
    tlch_about
    tlch_python
 
+.. toctree::
```

Comment:
IMHO, tutorials must be one of the first sections in the table of content, for example, after the overview.

Reply:
Agree